### PR TITLE
list bullet changing without animation

### PR DESCRIPTION
**Pull request by FaisalAbid**
The spec requires animation, which requires a bit of rework, what do you guys think about this?

If animation is important then I can work on that, no problem!

### CODE REVIEW

**Comment by kwalrath** [comment]
@FaisalAbid what should we be looking for?

**Comment by johnpryan** [comment]
Here's what I'm seeing:

4F35C339-B9DA-4FCC-B9D4-A33769FC80A1-78982-000331098AA2BFC1

1. I think the bullet is slightly too tall when it's not focused - it should look more like a bullet point:

Screen Shot 2019-07-23 at 4 11 12 PM

2. Without the animation the design doesn't achieve the same effect, but I'm comfortable merging this in the short term.

**Comment by FaisalAbid** [comment]
Let me work on the animation part and see how long it will take

**Comment by kwalrath** [comment]
@FaisalAbid can you update us on the status?

**Comment by FaisalAbid** [comment]
Hey, I'll get this done this end of this week

**Comment by FaisalAbid** [comment]
Hiya! I have a working version of this, cleaning it up and sending a PR by tomorrow

**Comment by FaisalAbid** [comment]
Hey @kwalrath I have the animated bullet points here. 

Let me know if this is acceptable, I have another very different approach im working on but I want to get your thoughts on this and see if this works!

**Comment by kwalrath** [comment]
@johnpryan is our resident CSS expert, so he's going to take a look.

**Comment by johnpryan** [comment]
@FaisalAbid when I load this in Chrome, I don't see the bullets animating - I'm wondering if using a GIF for the list-style-image is the best way. What was the other approach you had in mind?

**Comment by johnpryan** [comment]
Maybe something like [this](https://dartpad.dev/1659f19881f42816cc04f280b0c72d15) would work? I'm not sure if it would work with a list bullet though...

**Comment by FaisalAbid** [comment]
Hey @johnpryan - I think because Im referencing the files from JS, and the jekyl builder changes the file names, its not loading when its built, but works under serve, so forget that solution

Your solution is what I had in mind, basically get rid of the bullet points, and make a psuedo bullet with a css hover animation on it.

A bit messy and not following the standards but its the best solution I think.

I'll make this change

**Comment by FaisalAbid** [comment]
I'll wrap this up with the new css approach this week

**Comment by kwalrath** [comment]
> I'll wrap this up with the new css approach this week

Looking forward to seeing it!

**Comment by kwalrath** [comment]
@FaisalAbid how's the implementation going?

**Comment by FaisalAbid** [comment]
Have free time tomorrow -> friday, will get it done!

**Comment by FaisalAbid** [comment]
Hey, got this working, will send a PR shortly, had to refactor a few things on the home page for this change to keep the accessibility working, cleaning that up and will send the pr

**Comment by kwalrath** [comment]
Any update on the status, @FaisalAbid?

**Comment by FaisalAbid** [comment]
Sorry about that, got sidetracked by life, i'll update this with the latest code by tomorrow at the latest.

**Comment by FaisalAbid** [comment]
Highlight video in action https://youtu.be/L6VSfw-i6sY

@kwalrath @johnpryan  - do you want this effect for mobile as well, or do you want just static dot for mobile?

let me know and I'll make the change

**Comment by kwalrath** [comment]
That looks nice! I'll let @johnpryan answer your question.

**Comment by johnpryan** [comment]
This looks great! I think it's safe to do the animation on mobile devices too.

**Comment by FaisalAbid** [comment]
Cool, making the change for mobile, stay tuned!

**Comment by FaisalAbid** [comment]
Done! I fixed some padding and margin issues on mobile and desktop as well. Please give it a try on your end

File: src/_assets/css/_dash.scss [hunk 1]
```diff
@@ -149,8 +155,9 @@ $dash-dartpad-border: #293542;
       }
 
       ul li {
-        list-style-image: url(../../assets/dash/2x/bullet-black.png);
+        //list-style-image: url(../../assets/dash/2x/bullet-black.png);
```
Comment (johnpryan) [inline src/_assets/css/_dash.scss]: nit: remove commented out code

File: src/_assets/css/_dash.scss [hunk 2]
```diff
@@ -134,10 +134,16 @@ $dash-dartpad-border: #293542;
...
 
       .highlight {
         color: $dash-highlight;
-        list-style-image: url(../../assets/dash/2x/bullet.png);
+
+        .animated-bullet {
+          height: 25px;
+          background-color: $dash-highlight;
+        }
+        //list-style-image: url(../../assets/dash/2x/bullet.png);
```
Comment (johnpryan) [inline src/_assets/css/_dash.scss]: nit: commented out code

File: src/_assets/css/_dash.scss [hunk 3]
```diff
@@ -259,7 +284,8 @@ $dash-dartpad-border: #293542;
   }
 
   .selector{
-    margin-left:-24px;
+    margin-left:-30px;
+
```
Comment (johnpryan) [inline src/_assets/css/_dash.scss]: nit: spacing

File: src/_assets/css/_dash.scss (hunk 2 above)
Comment (FaisalAbid) [reply src/_assets/css/_dash.scss]: removed

File: src/_assets/css/_dash.scss (hunk 1 above)
Comment (FaisalAbid) [reply src/_assets/css/_dash.scss]: removed

File: src/_assets/css/_dash.scss (hunk 3 above)
Comment (FaisalAbid) [reply src/_assets/css/_dash.scss]: fixed

**Pull request merged**